lib/pf/ip4log.pm

```diff
@@ -67,7 +67,7 @@ sub ip2mac {
     }
 
     # We first query OMAPI since it is the fastest way and more reliable source of info in most cases
-    if ( isenabled($Config{omapi}{ip2mac_lookup}) ) {
+    if ( isenabled($Config{omapi}{ip2mac_lookup}) && isenabled($Config{'services'}{'dhcpd'}) ) {
```

Comment:
dhcpd doesn't exist anymore so that should be removed ?

At the same time the OMAPI code should also be removed